uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/CheckoutActivity.kt

```diff
@@ -50,10 +51,17 @@ class CheckoutActivity : BaseActivity(), WebViewActions {
         }
     }
 
+    fun removeIfCheckoutFragmentExists(){
+        for(item in supportFragmentManager.fragments){
+            if(item is CheckoutFragment){
+                supportFragmentManager.beginTransaction().remove(item).commit();
+            }
+        }
+    }
+
     override fun handleExtras() {
         // do nothing
     }
-
```

Comment:
Just a very minor thing, seems a bit of a random removal of a line here. Would be nice to keep it spaced out.